qwc-front.php

```diff
@@ -60,6 +60,27 @@ function qwc_add_filters_front() {
 	//foreach ( $url_filters as $name => $priority ) {
 	//	add_filter( $name, 'qtranxf_convertURL', $priority );
 	//}
+
+	/* Fix the product categories and tags (displayed above the "additional informations" tab) */
+	add_filter( 'wp_get_object_terms', function ( $terms ) {
+		foreach ( $terms as $term ) {
+			if ( $term->taxonomy == 'product_cat' || $term->taxonomy == 'product_tag' ) {
+				$term->name = qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage( $term->name );
+			}
+		}
+
+		return $terms;
+	} );
+
+	/* Fix the product attributes (displayed in the "additional informations" tab) */
+	add_filter( 'woocommerce_attribute', function ( $text ) {
+		$values = explode( ', ', $text );
+		foreach ( $values as $i => $val ) {
+			$values[ $i ] = qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage( $val );
+		}
+
+		return implode( ', ', $values );
+	} );
 }
 qwc_add_filters_front();
```

Comment:
Yes, I already had this migration problem with post meta and options, look at `function qtranxf_convert_database_postmeta`, for example how it is solved.

But I cannot believe, it does not work with normal `qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage`. Something is wrong, It should work, ...

There is a place where line breaks needed to be preserved and I introduced 'encode' => 'byline', but it does not seem to be applicable here, does it?